vendor/k8s.io/code-generator/Godeps/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
I thought `./hack/update-deps.sh` deletes `OWNERS` files in `vendor/`.

Reply:
If you run it ;-)

Reply:
Yep, if you run it applies :)